test/extended/testdata/cmd/test/cmd/newapp.sh

```diff
@@ -480,9 +458,9 @@ os::cmd::expect_success 'oc delete all -l app=testapp1'
 os::cmd::expect_success 'oc delete all -l app=ruby --ignore-not-found'
 os::cmd::expect_success 'oc delete imagestreams --all --ignore-not-found'
 # newapp does not attempt to create an imagestream that already exists for a container image
-os::cmd::expect_success 'oc new-app docker.io/ruby:2.2'
+os::cmd::expect_success 'oc new-app docker.io/ruby:2.5'
```

Comment:
doesn't this still use docker.io?

Reply:
It does :( There are a few sill left, but I can't find anything to replace that temporarily. I'll return to this after this merges.

Reply:
Also newapp.sh is on the blacklist https://github.com/openshift/origin/blob/ee82ae0d236efb69590dd4ed5ad29452eb62a4a0/test/extended/cmd/cmd.go#L28-L34 so it's not executed.

Reply:
language nit - we should rename this var to `skipList`. The usage of blacklist/whitelist is discouraged.

Reply:
This will disappear as soon as I'm done with all the migration.